nservicebus/hosting/nservicebus-host/index.md

```diff
@@ -25,6 +25,15 @@ If you want to avoid the scanning process you can explicitly configure the type
 snippet:ExplicitHostConfigType
 
 
+### Controlling assembly scanning using the command line
+
+A list of assemblies to scan can also be controlled using the `/scannedAssemblies` switch. If this option is used, the `NServiceBus.Host.exe` loads only assemblies (with references) that have been explicitly listed on the command line. Each assembly must be added using a separate switch:
+
+`NServiceBus.Host.exe /scannedAssemblies:"NServiceBus.Host" /scannedAssemblies:"MyMessages" /scannedAssemblies:"MyEndpoint"`
+
+NOTE: The `NServiceBus.Host` assembly must be listed explicitly. In this case `NServiceBus.Core` is loaded as a reference and need not be specified separately.
```

Comment:
The assembly scanning doco says:

>   NOTE: During the scanning process, the core dlls for NServiceBus namely `NServiceBus.Core.dll`, `NServiceBus.dll` (in versions prior to Version 5) and if in use `NServiceBus.Host.exe` are automatically included since the endpoint needs them to function properly.

Any reason why the NServiceBus.Host needs to be specified explicitly in the command line argument? If so can that reason be clarified as well?